### PR TITLE
Polish en APIs of unique_name

### PR DESCRIPTION
**Pull request by zhiqiu**
This PR updates en document of three APIs, which are fluid.unique_name.generate(),  fluid.unique_name.guard() and  fluid.unique_name.switch().

### CODE REVIEW

**Comment by zhiqiu** [comment]
Corresponding Chinese APIs, [PR1402](https://github.com/PaddlePaddle/FluidDoc/pull/1402).

image


image

image

**Review by sneaxiy — COMMENTED** [review]
Try to use the newest preview tools to show whether the style of `Return type` is right.

**Comment by zhiqiu** [comment]
generate()
image

guard()
image
image


switch()
image
image

**Review by sneaxiy — DISMISSED** [review]
LGTM.

File: python/paddle/fluid/unique_name.py [hunk 1]
```diff
@@ -102,27 +103,31 @@ def generate_with_ignorable_key(key):
...
         .. code-block:: python
 
             import paddle.fluid as fluid
             name1 = fluid.unique_name.generate('fc')
             name2 = fluid.unique_name.generate('fc')
-            # The result is fc_0, fc_1
-            print name1, name2 
+            print(name1, name2) # fc_0, fc_1
 
             fluid.unique_name.switch()
```
Comment (lanxianghit) [inline python/paddle/fluid/unique_name.py]: 最好可以添加一个参数不为None的例子

File: python/paddle/fluid/unique_name.py [hunk 2]
```diff
@@ -136,30 +141,35 @@ def switch(new_generator=None):
...
+
+    Examples: 
 
-    Examples:
         .. code-block:: python
 
             import paddle.fluid as fluid
             with fluid.unique_name.guard():
               name_1 = fluid.unique_name.generate('fc')
             with fluid.unique_name.guard():
```
Comment (lanxianghit) [inline python/paddle/fluid/unique_name.py:169]: 最好给一个带参数时的例子

**Review by lanxianghit — DISMISSED** [review]

File: python/paddle/fluid/unique_name.py (hunk 2 above)
Comment (zhiqiu) [reply python/paddle/fluid/unique_name.py:169]: Provided below, thanks.

File: python/paddle/fluid/unique_name.py (hunk 1 above)
Comment (zhiqiu) [reply python/paddle/fluid/unique_name.py]: Done, thanks.

**Comment by zhiqiu** [comment]
follow comments
image
image

**Review by sneaxiy — DISMISSED** [review]

**Review by lanxianghit — DISMISSED** [review]
LGTM

**Review by lanxianghit — DISMISSED** [review]

**Review by XiaoguangHu01 — DISMISSED** [review]

**Review by lanxianghit — APPROVED** [review]

**Review by wanghaoshuang — APPROVED** [review]

**Pull request merged**